tests/test-recipes/split-packages/_test-file-hash/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: _file_hash-split
+  version: 0.0.1
+
+build:
+  number: 0
+  # skip: True  # [not linux64]
```

Comment:
Not sure if we only want it on linux or all platforms. Commenting it out for now and then we see if we can leave it in or out.

Reply:
```suggestion
  skip: True  # [not linux64]
```